core/variant/variant.cpp

```diff
@@ -1664,7 +1664,7 @@ String Variant::stringify(int recursion_count) const {
 		case INT:
 			return itos(_data._int);
 		case FLOAT:
-			return rtos(_data._float);
+			return String::num_real(_data._float, true);
```

Comment:
`rtos` is designed to print out doubles (14 digits), while `String::num_real` is designed to print out `real_t`s (either 6 or 14 digits) for the types like Vector3. As-is this change isn't good for Variant because it uses double.

Reply:
I changed how this method is implemented. You need to pass actual `real_t` to get the old behavior.